### PR TITLE
added some seo friendly meta tags to website

### PR DESCRIPTION
**Pull request by coderelay2020**
this is just a small thing related to SEO there is more to it

### CODE REVIEW

**Comment by coderelay2020** [comment]
#46 issue - added some seo friendly meta tags to website

**Review by jeffslofish — CHANGES_REQUESTED** [review]
Hi @coderelay2020, I am getting this error when I run the code:


Request from 127.0.0.1: GET /assets/icons/favicon.ico
Response with status 500 in 1 ms.
Error during invocation:  Error: Cannot find module '/Users/jeffreyfisher/Code/open-source-help-wanted/functions/assets'
Require stack:
- /Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/netlify-lambda/lib/serve.js
- /Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/netlify-lambda/bin/cmd.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at /Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/netlify-lambda/lib/serve.js:122:17
    at Layer.handle [as handle_request] (/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/express/lib/router/route.js:144:13)
    at next (/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/express/lib/router/route.js:138:14)
    at next (/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/express/lib/router/route.js:138:14)
    at next (/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/express/lib/router/route.js:138:14)
    at next (/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/express/lib/router/route.js:138:14)
    at next (/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/express/lib/router/route.js:138:14)
    at next (/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/express/lib/router/route.js:138:14)
    at Route.dispatch (/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/express/lib/router/layer.js:95:5)
    at /Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/express/lib/router/index.js:284:15 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/netlify-lambda/lib/serve.js',
    '/Users/jeffreyfisher/Code/open-source-help-wanted/node_modules/netlify-lambda/bin/cmd.js'
  ]
}

And I get this in the console:

GET http://localhost:3000/assets/icons/favicon.ico 500 (Internal Server Error)

File: src/App.tsx [hunk 1]
```diff
@@ -11,6 +12,33 @@ ReactGA.send('pageview');
 const App = () => {
   return (
     <Router>
+    <Helmet>
+
+          <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+          <meta property="og:type" content="website" />
+          <meta name="twitter:site" content="@teladochealth" />
```
[the remark this answers is not in the document]
Comment (coderelay2020) [reply src/App.tsx]: I have resolved the issue.
I updated locally but forgot push sorry for the inconvenience

**Comment by jeffslofish** [comment]
@coderelay2020 Thank you, it is now merged!

**Pull request merged**